server/service/activities.go

```diff
@@ -38,3 +38,12 @@ func (svc *Service) ListActivities(ctx context.Context, opt fleet.ListOptions) (
 	}
 	return svc.ds.ListActivities(ctx, opt)
 }
+
+func (svc *Service) GenerateActivity(ctx context.Context, user *fleet.User, activityType string, details *map[string]interface{}) error {
+	if svc.EnterpriseOverrides != nil && svc.EnterpriseOverrides.GenerateActivity != nil {
+		return svc.EnterpriseOverrides.GenerateActivity(ctx, user, activityType, details)
```

Comment:
Generate vs CreateActivity... any reason why using different names?

Reply:
- `Service.GenerateActivity` creates the activity in Fleet and then does other stuff like streaming it to external locations (vs `fleet.CreateActivity` which just creates the activity in a Fleet `Datastore`).